pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -127,7 +126,7 @@ const CryptoPolicyDialog = ({
             .filter(pol => pol.endsWith(':SHA1') ? shaSubPolicyAvailable : true)
             .map(policy => ({
                 name: policy,
-                title: displayProfileText(policy),
+                title: policy,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18907-20230623-160035-2a44ca96-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)